docs/mp/config/01_introduction.adoc

```diff
@@ -43,29 +81,88 @@ Step-by-step guide about using {spec-name} in your Helidon MP application.
 --
 ====
 
-== Using {spec-name} in Helidon MP applications
 
-=== Configuration
+=== Using MicroProfile Config Sources and META-INF
+
+Default configuration values can be retrieved from system properties, environment variables, files, directories, URLs or configuration files of the form `${CLASSPATH}/META-INF/microprofile-config.properties`. 
+
+The example below shows how the MicroProfile configuration file `microprofile-config.properties` can be used to modify properties such as server port or host IP. 
+
+----
+# Application properties. This is the default greeting
+app.greeting=Hello
+
+# Microprofile server properties
+server.port=8080
+server.host=0.0.0.0
+----
+
+=== Using MicroProfile Config API
+
+You can use the default MicroProfile configuration by injecting configuration properties or use `ConfigProvider.getConfig()`.
+
+----
+   org.eclipse.microprofile.config.Config config = ConfigProvider.getConfig();
+----
+
+=== Using Helidon Meta Configuration
+
+The meta configuration uses underlying implementation of Helidon config. In this case, you can either inject configuration properties or use `ConfigProvider.getConfig()`.
```

Comment:
"In this case"? Which is the same as above - so the second sentence is I think redundant.